geometry_msgs/Point p
geometry_msgs/Point dir
string object
string mesh_resource
---
bool success
